### DISCUSSION TITLE
Fail code

### DISCUSSION THREAD

**Discussion by Coral-Inc** [community/community#51062] [Other Feature Feedback, Questions, & Ideas] [answered]
Everytime i try link to another folders file it fails. I tried everything even with / \. Can anyone search my code and help me out? Link: [Repository Link](https://github.com/Coral-Inc/beta) (Only search in folder v.0.1.4, or else its loose of time)

**Comment by suraj719** [comment]
hey, try linking the file with ../ for example if u want to access style.css (which is in css folder) in index.html(in home folder) then you may use ../css/style.css . Let me know if it doesn't work.

**Reply by Coral-Inc** [reply]
Hello. It might be me making a mistake. But it doesnt seem to work 😬

**Comment by enviGit** [comment, marked as answer]
### Hi [Sebastian](https://github.com/Coral-Inc)!

It seems that you are trying to access files from directories that are not in the same folder as your HTML files, and the `../` notation is used to go **up one directory level** from the **current directory**.

If your HTML file for example is located in a directory called `Home`, and you want to access an image located in a directory called `Pic` at the same level as `Home`, you can use the following code to reference the image file:
`<img src="../Pic/Copy of Coral Adalo Logo - Made with PosterMyWall.png" style="width: 125px; float: left;">` //This one is correct
`<img src="../v.0.1.4/Pic/Copy of Coral Adalo Logo - Made with PosterMyWall.png" style="width: 125px; float: left;">` //This is the current one

In this example, the `../` indicates that we are going up one level from the current directory (`Home`) to access the `Pic` directory, and then we specify the name of the image file we want to include. Basically you need to remove `/v.0.1.4` part from your paths

Generally speaking a relative path is a file path that is specified relative to the current file's location, instead of specifying the complete path from the root directory. So in your current code, the path `../v.0.1.4/Pic/Copy of Coral Adalo Logo - Made with PosterMyWall.png` is a valid relative path **ONLY** if the file you are linking from is located in a folder that is one level up from the `v.0.1.4` folder (`beta` folder in this case).

**Reply by Coral-Inc** [reply]
Thank you!
